include/pika_geohash.h

```diff
@@ -32,26 +32,26 @@
 #ifndef PIKA_GEOHASH_H_
 #define PIKA_GEOHASH_H_
 
-#include <stddef.h>
-#include <stdint.h>
+#include <cstddef>
+#include <cstdint>
 
 #if defined(__cplusplus)
 extern "C" {
 #endif
 
 #define HASHISZERO(r) (!(r).bits && !(r).step)
 #define RANGEISZERO(r) (!(r).max && !(r).min)
-#define RANGEPISZERO(r) (r == nullptr || RANGEISZERO(*r))
+#define RANGEPISZERO(r) ((r) == nullptr || RANGEISZERO(*(r)))
 
 #define GEO_STEP_MAX 26 /* 26*2 = 52 bits. */
 
 /* Limits from EPSG:900913 / EPSG:3785 / OSGEO:41001 */
-#define GEO_LAT_MIN -85.05112878
+#define GEO_LAT_MIN (-85.05112878)
```

Comment:
这里常量建议用 const double （如果支持constexpr就用这个）
```suggestion
const dobule GEO_LAT_MIN = -85.05112878;
```

Reply:
很好的建议，我来改下